doc/apiv3-documentation.apib

```diff
@@ -3390,41 +3436,306 @@ For more details and all possible responses see the general specification of [Fo
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:MissingPermission",
+                "message": "You are not allowed to add watchers to this work package."
+            }
+
++ Response 404 (application/hal+json)
+
+    Returned if the work package does not exist or the client does not have sufficient permissions to see it.
+
+    **Required permission:** view work package
+
+    *Note that you will effectively not be able to change the watchers of a work package without being able to see the work package.*
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:NotFound",
+                "message": "The specified work package does not exist."
+            }
+
++ Response 422 (application/hal+json)
+
+    Returned if:
+
+    * the client tries to specify a link to a resource that is not a user (`ResourceTypeMismatch`)
+    * the user specified is not allowed to watch that work package (`PropertyConstraintViolation`)
+    * the user specified does not exist (`PropertyConstraintViolation`)
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:ResourceTypeMismatch",
+                "message": "Expected resource of type 'User', but got a 'Status'.",
+            }
+
 ## Remove Watcher [/api/v3/work_packages/{work_package_id}/watchers/{id}]
 
 ## Remove watcher [DELETE]
 
+Removes the specified user from the list of watchers for the given work package.
+
+If the request succeeds, the specified user is not watching the work package anymore.
+
+*Note: This might also be the case, if the specified user did not watch the work package prior to the request.*
+
 + Parameters
     + work_package_id (required, integer, `1`) ... Work package id
     + id (required, integer, `1`) ... User id
 
 + Response 204
 
-## Available Watchers [/api/v3/work_packages/{work_package_id}/available_watchers]
++ Response 403 (application/hal+json)
+
+    Returned if the client does not have sufficient permissions.
+
+    **Required permission:** delete work package watchers
+
+    *Note that you will only receive this error, if you are at least allowed to see the corresponding work package.*
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:MissingPermission",
+                "message": "You are not allowed to remove watchers from this work package."
+            }
+
++ Response 404 (application/hal+json)
+
+    Returned in one of two cases:
+
+    Either the work package does not exist or the client does not have sufficient permissions to see it.
+
+    **Required permission:** view work package
+
+    Or the specified user does not exist at all.
```

Comment:
That would be a third case, you might to consider rephrasing your sentence above which states that there are only two cases. Also: Maybe pull this sentence, or its statement for that matter, above the line with Required permission.